home-screen-widget/profiles/JS_SDK.md

```diff
@@ -0,0 +1,15 @@
+### Javascript SDK API
```

Comment:
Do we even need this page? Don't we already have one like this in the developers portal?

Reply:
I thought the one we have there is the Java one while this is the javascript one, that's why I thought I'd add it.

Reply:
Yes, there's documentation for the Java one. If we don't yet have public documentation for the Javascript Beekeeper SDK then we should probably spend a lot more time on writing that up, now that we're placing it so prominently with the home screen widgets.

Reply:
ok cool, I'll create a follow up ticket for that